lambda/security-groups/src/main/scala/com/gu/hq/Notifier.scala

```diff
@@ -33,11 +33,11 @@ object Notifier extends StrictLogging {
     }
   }
 
-  def createNotification(groupId: String, targetTags: List[Tag], accountId: String, regionName: String): Notification = {
+  def createNotification(groupId: String, targetTags: List[Tag], accountId: String, accountName:String, regionName: String): Notification = {
     val actions = List(
       Action("View in AWS Console", s"https://$regionName.console.aws.amazon.com/ec2/v2/home?region=$regionName#SecurityGroups:search=$groupId")
     )
-    val message = s"Warning: Security group '$groupId' is open to the world"
+    val message = s"Warning: Security group '$groupId' in account '$accountName' is open to the world"
```

Comment:
Minor point / bikeshed

Maybe check the formatting Anghammarad supports? Can we do better than quoted?

Email supports markdown, so bold might be nice. I can't remember if Google Hangouts doesn't support much formatting in cards, but IIRC bold is fine. The Anghammarad code shows the transformations it does.

Reply:
Summary: I am basically saying I think `*string*` will display as bold on both emails and chat messages..

Reply:
https://github.com/guardian/anghammarad/blob/master/anghammarad/src/main/scala/com/gu/anghammarad/messages/Messages.scala#L51

The message is markdown, these are the fixes Anghammarad does because Google Hangouts Chat only supports a weird subset of tags.

Reply:
Done.